css/theme-weebly.scss

```diff
@@ -16,46 +16,46 @@ $font-size-sm: 13px;
 $font-size-xs: 12px;
 
 $border-color: #dfe6ee;
-$border-radius: 6px;
+$border-radius: 2px;
 $border-radius-sm: 4px;
 
-$page-max-width: 972px;
+$page-max-width: 1024px;
 $sidebar-width: 220px;
 
 .nav-tabs > li.active > a,
 .nav-tabs > li.active > a:hover,
 .nav-tabs > li.active > a:focus {
-    background-color: #248ec2;
-    color: white !important;
+	background-color: #248ec2;
+	color: white !important;
 }
 
 div.navbar-collapse .dropdown-menu > li > a:hover {
-    background-color: #347DBE;
```

Comment:
More tab reverts plz.

Reply:
Can you add the `.editorconfig` file at the root of the repo. Won't have to worry about tabs v spaces any more.